packages/paste-core/components/tooltip/stories/index.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import {Absolute} from '@twilio-paste/absolute';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {Text} from '@twilio-paste/text';
+import {Tooltip} from '../src';
+
+const Example: React.FC<{}> = () => {
+  return (
+    <Tooltip text="Welcome to Paste!">
+      <Button variant="primary">Open Tooltip</Button>
+    </Tooltip>
+  );
+};
+
+const IconButtonExample: React.FC<{}> = () => {
+  return (
+    <Tooltip text="Welcome to Paste!">
+      <Button variant="secondary" size="icon">
+        <InformationIcon decorative={false} title="Open Tooltip" />
+      </Button>
+    </Tooltip>
+  );
+};
+
+const CustomExample: React.FC<{}> = () => {
+  return (
+    <Box display="flex" alignItems="center">
+      <Text as="span">Tooltip should appear from the icon.</Text>
+      <Tooltip text="Welcome to Paste!">
+        <a href="https://paste.twilio.com" style={{display: 'inline-flex'}}>
+          <InformationIcon decorative={false} title="Open Tooltip" />
+        </a>
+      </Tooltip>
+    </Box>
+  );
+};
+
+const BottomExample: React.FC<{}> = () => {
+  return (
+    <Tooltip text="Welcome to Paste!" placement="bottom">
+      <Button variant="primary">Open Tooltip</Button>
+    </Tooltip>
+  );
+};
+
+const TopExample: React.FC<{}> = () => {
+  return (
+    <Absolute preset="bottom" bottom={12}>
+      <Tooltip text="Welcome to Paste!" placement="top-start">
+        <Button variant="primary">Open Tooltip</Button>
+      </Tooltip>
+    </Absolute>
+  );
+};
+
+const RightExample: React.FC<{}> = () => {
+  return (
+    <Tooltip text="Welcome to Paste!" placement="right-start">
+      <Button variant="primary">Open Tooltip</Button>
+    </Tooltip>
+  );
+};
+
+const LeftExample: React.FC<{}> = () => {
+  return (
+    <Absolute preset="right" top={12} right={12}>
+      <Tooltip text="Welcome to Paste!" placement="left-start">
+        <Button variant="primary">Open Tooltip</Button>
+      </Tooltip>
+    </Absolute>
+  );
+};
+
+storiesOf('Components|Tooltip', module)
+  .addDecorator(withKnobs)
+  .add('Default', () => {
+    return <Example />;
```

Comment:
Would be nice to pass a knob here to toggle the placement and text

Reply:
I passed a text knob. I tried the placement one, but it doesn't want to change when the knob is changed. Not unless I refresh and change it manually. Maybe because its a hook in the component?